src/components/footer/footer.css

```diff
@@ -2,64 +2,74 @@
 
 :host {
   .copyright-text {
-    margin: 10px auto;
+    margin: 0 auto;
+    padding: 0;
     text-align: center;
     color: var(--color-dark);
     text-transform: uppercase;
     font-family: var(--font-family-primary);
   }
-}
-/* W3C headlines on footer h2 to h3 nested headlines */
-footer > section > h2,
-footer > section > article > h3 {
-  display: none;
-}
-
-article:nth-of-type(2) {
-  ul.socials {
-    /* keeps it centered */
-    display: flex;
-    justify-content: center;
+  section > h2,
+  section > article > h3 {
+    display: none;
+  }
+  section > article {
+    margin: 0 auto;
     padding: 0;
-    li {
-      list-style-type: none;
-      /* creates spacing between icons */
-      padding: 0% 0.75%;
-      strong {
-        display: none;
-      }
-      a {
-        i {
-          font-size: 2rem;
-          color: var(--color-dark);
-        }
-        i:hover {
-          color: var(--color-tertiary);
+  }
+  .col-xs-12:nth-of-type(1) {
+    float: none;
+  }
+  article:nth-of-type(1) {
+    margin: 1.5% 0% 0% 0%;
+    padding: 0;
+    ul.socials {
+      display: flex;
+      justify-content: center;
+      margin: 0 auto;
+      padding: 0;
+      li {
+        list-style-type: none;
+        padding: 0% 0.75% 0% 0%;
+        strong {
+          display: none;
         }
-      }
-
-      @media screen and (max-width: 900px) {
         a {
           i {
-            font-size: 2rem !important;
+            font-size: 3rem;
```

Comment:
https://github.com/AnalogStudiosRI/www.analogstudios.net/pull/60#discussion_r778207024

Reference to making the icons a bit bigger to 3rem.